packages/router-plugin/tests/router.plugin.spec.ts

```diff
@@ -23,70 +12,19 @@ import {
   ofActionSuccessful,
   State,
   Action,
-  StateContext,
-  Select,
-  Selector
+  StateContext
 } from '@ngxs/store';
 
 import {
   NgxsRouterPluginModule,
   RouterState,
   RouterStateSerializer,
   Navigate,
-  RouterNavigation,
-  RouterStateModel,
-  RouterDataResolved
+  RouterNavigation
 } from '../';
 
 describe('NgxsRouterPlugin', () => {
-  it('should dispatch router state events', fakeAsync(async () => {
```

Comment:
Shouldn't we restore this test?

Reply:
I'm not sure. This test is not explicit and we always have to fix it when we do any change. I don't think it's a good approach.

Reply:
I think we could restore this test but run expectations on other things, not logs.

Reply:
This type of test is often called a characterization test. It is not necessarily saying that the behavior is right or wrong but rather just documenting what the current behavior is. The value here is to know what has changed when it does.
You could suffix the test name with "(Characterization)" to make its value explicit.

Reply:
I will move it to another file for this type of tests to avoid mess.